server/src/utils/middlewares/errorHandlerMiddleware.ts

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright 2022 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ *  renku-ui-server
+ *
+ *  errorHandlerMiddleware.ts
+ *  errorHandler middleware
+ */
+
+import { BaseError } from "../baseError";
+import logger from "../../logger";
+import { errorHandler } from "../errorHandler";
+import express from "express";
+
+const errorHandlerMiddleware = (
+  err: Error,
+  req: express.Request,
+  res: express.Response,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  next: express.NextFunction): void => {
```

Comment:
If you are not invoking `next`, you can just leave it out.
```suggestion
  res: express.Response): void => {
```

Reply:
The error handler doesn't work if the `next` is not specified as a parameter. 🤷‍♀️